plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/OptInDialog.java

```diff
@@ -36,18 +34,12 @@
  */
 public class OptInDialog extends Dialog {
```

Comment:
Would be worth adding javadoc to indicate that SHARE &rarr; `Window.OK` and DO_NOT_SHARE &rarr; `Window.CANCEL`.

Reply:
Will add the Javadoc.